commit 5540f4329bbdb4deb4850d4af48b18ad074bba19
Author: Jia Tan <jiat0218@gmail.com>
Date:   Fri Jan 19 23:08:14 2024 +0800

    Docs: Update .xz file format specification to 1.2.0.
    
    The new RISC-V filter was added to the specification, in addition to
    updating the specification URL.

diff --git a/doc/xz-file-format.txt b/doc/xz-file-format.txt
index 09c83e0c..2a018657 100644
--- a/doc/xz-file-format.txt
+++ b/doc/xz-file-format.txt
@@ -2,7 +2,7 @@
 The .xz File Format
 ===================
 
-Version 1.1.0 (2022-12-11)
+Version 1.2.0 (2024-01-19)
 
 
         0. Preface
@@ -81,18 +81,22 @@ Version 1.1.0 (2022-12-11)
 0.2. Getting the Latest Version
 
         The latest official version of this document can be downloaded
-        from <http://tukaani.org/xz/xz-file-format.txt>.
+        from <https://xz.tukaani.org/format/xz-file-format.txt>.
 
         Specific versions of this document have a filename
         xz-file-format-X.Y.Z.txt where X.Y.Z is the version number.
         For example, the version 1.0.0 of this document is available
-        at <http://tukaani.org/xz/xz-file-format-1.0.0.txt>.
+        at <https://xz.tukaani.org/format/xz-file-format-1.0.0.txt>.
 
 
 0.3. Version History
 
         Version   Date          Description
 
+        1.2.0     2024-01-19    Added RISC-V filter and updated URLs in
+                                Sections 0.2 and 7. The URL of this
+                                specification was changed.
+
         1.1.0     2022-12-11    Added ARM64 filter and clarified 32-bit
                                 ARM endianness in Section 5.3.2,
                                 language improvements in Section 5.4
@@ -923,6 +927,7 @@ Version 1.1.0 (2022-12-11)
               0x08       2 bytes    ARM Thumb filter [1]
               0x09       4 bytes    SPARC filter
               0x0A       4 bytes    ARM64 filter [2]
+              0x0B       2 bytes    RISC-V filter
 
               [1] These are for little endian instruction encoding.
                   This must not be confused with data endianness.
@@ -1136,30 +1141,30 @@ Version 1.1.0 (2022-12-11)
 7. References
 
         LZMA SDK - The original LZMA implementation
-        http://7-zip.org/sdk.html
+        https://7-zip.org/sdk.html
 
         LZMA Utils - LZMA adapted to POSIX-like systems
-        http://tukaani.org/lzma/
+        https://tukaani.org/lzma/
 
         XZ Utils - The next generation of LZMA Utils
-        http://tukaani.org/xz/
+        https://xz.tukaani.org/xz-utils/
 
         [RFC-1952]
         GZIP file format specification version 4.3
-        http://www.ietf.org/rfc/rfc1952.txt
+        https://www.ietf.org/rfc/rfc1952.txt
           - Notation of byte boxes in section "2.1. Overall conventions"
 
         [RFC-2119]
         Key words for use in RFCs to Indicate Requirement Levels
-        http://www.ietf.org/rfc/rfc2119.txt
+        https://www.ietf.org/rfc/rfc2119.txt
 
         [GNU-tar]
-        GNU tar 1.21 manual
-        http://www.gnu.org/software/tar/manual/html_node/Blocking-Factor.html
+        GNU tar 1.35 manual
+        https://www.gnu.org/software/tar/manual/html_node/Blocking-Factor.html
           - Node 9.4.2 "Blocking Factor", paragraph that begins
             "gzip will complain about trailing garbage"
           - Note that this URL points to the latest version of the
             manual, and may some day not contain the note which is in
-            1.21. For the exact version of the manual, download GNU
-            tar 1.21: ftp://ftp.gnu.org/pub/gnu/tar/tar-1.21.tar.gz
+            1.35. For the exact version of the manual, download GNU
+            tar 1.35: ftp://ftp.gnu.org/pub/gnu/tar/tar-1.35.tar.gz
 